test/python/test_ngraph_serialize_flag.py

```diff
@@ -46,13 +46,10 @@ def test_ng_serialize_to_json(self):
         out = tf.nn.l2_loss(tf.abs(x))
         values = np.random.rand(*xshape)
 
-        config = ngraph_bridge.update_config(tf.compat.v1.ConfigProto())
         ngraph_enable_serialize = os.environ.pop('NGRAPH_ENABLE_SERIALIZE',
                                                  None)
         os.environ['NGRAPH_ENABLE_SERIALIZE'] = '1'
-        ngraph_bridge.enable()
```

Comment:
How are these changes related to increasing minimum #of trivial nodes?

Reply:
This test wasn't written to run with cluster de-assignment disabled, and hence depends on the parameters used for cluster deassignment.